backend/lm_backend/api/cruds/generic.py

```diff
@@ -174,3 +169,15 @@ async def delete(self, db_session: AsyncSession, id: Union[Column[int], int]):
             raise HTTPException(status_code=400, detail=f"{self.model.__name__} could not be deleted.")
 
         return {"message": f"{self.model.__name__} deleted successfully."}
+
+
+class GenericCRUD(_GenericCRUD):
```

Comment:
That's so weird, why is `mypy` complaining about this just now?

Reply:
I have no idea why this suddenly started.